packages/affiliates/liquidity-mining/yusdeth-weekly-payouts/Week_22_Mining_Rewards.json

```diff
@@ -0,0 +1,266 @@
+    "0xb8bdffa3de9939ced80769b0b9419746a49f7aa5": "19.706917877071005327",
+    "0xbd1f7d88c76a86c60d41bddd4819fae404e7151e": "0.806395217846690837",
+    "0xbddbc13abdc4d3b617c2e573c26cd5a7f24a0478": "0.041822307418415778",
+    "0xc10aeee4b362b8483fc97c06b628e82f04068fa4": "20.68432611956019832",
+    "0xc1cad6df277106222dd45cf5b0300fbd4d1193d5": "0.067100959660783887",
+    "0xc28e23e7d7d6cc0c9c6f08146c71e7d168cae45a": "0.1728071830346174",
+    "0xc45cdbd2e308228d409d625c8c3d94b70211becb": "0.040491057027887177",
+    "0xc58b20c01c1f24e72ee2c36a773ef48e774ea595": "0.045785791619822858",
+    "0xc61f0667076521761fb365f52644572e92fd0c94": "0.004017983558685611",
+    "0xc7777c1a0cf7e22c51b44f7ced65cf2a6b06dc5c": "1.142848512345964289",
+    "0xc7ca46dcc1ddaad78081b12f64bd61d9f0f2f22d": "0.191706330734015849",
+    "0xc806a568ea0c7adc17b40962776116a645d4ff76": "0.051060892226609652",
+    "0xcee2a8ca6f188f1b358d64986904ad11983f8531": "16.265138370130809519",
+    "0xd0dc07b98769f23a7bdbef15a35faa256cb65dcf": "0.228368913461593508",
+    "0xd165164cbab65004da73c596712687c16b981274": "4.031976089233474272",
+    "0xd27d90b337717fd91c4d409f712b890d4ee2fdd6": "10.445840418526223898",
+    "0xd2a78bb82389d30075144d17e782964918999f7f": "23.000480480887053178",
+    "0xd9e5a1ae7f035ac2d4cfb7084484e17d87993128": "34.466851106470238913",
+    "0xdb0db7bf65f3f6e6909a02f7f34ad0f669e5ec9e": "1.036622184123353686",
+    "0xdc7c5b169e17aef7dbc51e0a19eb9223a96b0193": "0.694681249462011678",
+    "0xde145f934f7e0f30f412a33a69cd089f57cec3b8": "1.161127519036619127",
+    "0xdf2583eaef0ed1992e503f06e37392dc2e344f20": "49.020123501676249464",
+    "0xdfa8ee9ff8798b37800f3ab18807273d7caaf1b6": "72.586405422784369317",
+    "0xe1c9acd032155c289e55f504c07f269aab70d439": "43.659115809275814473",
+    "0xe6662ec7fc7c84231b29f0f0da6321673d100854": "4.095306992985766009",
+    "0xe7da19d08df73c9371080a14b8e6fdca9243e16e": "105.184331406806054759",
+    "0xea32a33300d64bddb9f09a8308d44c23616ec706": "19.897947239773176214",
+    "0xec005b6edba4eb0a230317921555bc8d548d8224": "38.291876750412942107",
+    "0xec419ef9dd921c031b807e27a96f1518b9c449e3": "4.95922604657883819",
+    "0xed02f16a57c32a08b55d923bf4690d200722f462": "0.234505870870178531",
+    "0xef8305e140ac520225daf050e2f71d5fbcc543e7": "0.000571146191507135",
+    "0xf1ee1f3c5b4fc684d50b61c458341a73abd63b1d": "0.201764105820403436",
+    "0xf307077dd0e27a382e93f2e3d13a9c6584582332": "0.008263444275793229",
+    "0xf6f26015f0760f7a4b7f93e765bc48883daf79a5": "14.564231127968763209",
+    "0xf8a8d25049ebfaf36cf1dd7ff51ebd0777fc9b32": "40.859091614066124522",
+    "0xf94b3aa4415a14b289e329f9a75323656a2dcbc2": "2.324524535372499293",
+    "0xfc5be65e67bf016e414db1655133529033f0fa15": "0.455056809400005615",
+    "0xfdcfc1188005779ad1c70a0560fd5bbadaffebf5": "0.49548208469855303",
+    "0xffd44213718b48a371424303fe4aa2395ed4ce62": "1320.276063440137078706"
```

Comment:
this address exists in this pool as the second largest position `0xCce41676a4624F4A1e33a787A59d6bF96e5067bc` but does not exist in the second pool `0x5e065D534d1DAaf9E6222AfA1D09e7Dac6cbD0f7`, yet if i understand this right, the address earns UMA in both. Is this right?

Reply:
actually nevermind, i think i needed to refresh the page balance pool page between switching contracts 😩